### PR TITLE
Added fully-working SQLite database

### PR DESCRIPTION
**Pull request by Oleh325**

### CODE REVIEW

File: app/src/main/java/com/bbdevs/app/CatAppDB.kt [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    fun addTask(name : String, description : String,
+                reward : String, deadline : String ){
+        val values = ContentValues()
+
+        values.put(NAME, name)
+        values.put(DESCRIPTION, description)
+        values.put(REWARD, reward)
+        values.put(DEADLINE, deadline)
+        values.put(IS_COMPLETED, "0")
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/CatAppDB.kt]: "0"  to const

File: app/src/main/java/com/bbdevs/app/CatAppDB.kt [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+        db.close()
+    }
+
+    fun getStatistics(): Cursor? {
+        val db = this.readableDatabase
+        return db.rawQuery("SELECT * FROM statisticsPerDay_table", null)
+    }
+
+    fun addUserInfo(balance : String, cat_health : String ){
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/CatAppDB.kt:69]: space before {

File: app/src/main/java/com/bbdevs/app/CatAppDB.kt [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+        db.close()
+    }
+
+    fun getUserInfo(): Cursor? {
+        val db = this.readableDatabase
+        return db.rawQuery("SELECT * FROM userInfo_table", null)
+    }
+
+    companion object Tasks{
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/CatAppDB.kt:83]: space

File: app/src/main/java/com/bbdevs/app/MainActivity.kt [hunk 4]
```diff
@@ -11,60 +11,83 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/MainActivity.kt:12]: tabulation

File: app/src/main/java/com/bbdevs/app/MainActivity.kt [hunk 5]
```diff
@@ -11,60 +11,83 @@ class MainActivity : AppCompatActivity() {
...
+            val deadline = enterDeadline.text.toString()
 
-            // calling method to add
-            // name to our database
-            db.addName(name, age)
+            db.addTask(name, description, reward, deadline)
 
-            // Toast to message on the screen
-            Toast.makeText(this, name + " added to database", Toast.LENGTH_LONG).show()
+            Toast.makeText(this, "$name added to database", Toast.LENGTH_LONG).show()
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/MainActivity.kt:22]: does this name work?

File: app/src/main/java/com/bbdevs/app/MainActivity.kt [hunk 6]
```diff
@@ -11,60 +11,83 @@ class MainActivity : AppCompatActivity() {
...
 
-            // at last, clearing edit texts
             enterName.text.clear()
-            enterAge.text.clear()
+            enterDescription.text.clear()
+            enterReward.text.clear()
+            enterDeadline.text.clear()
+        }
+
```
Comment (BogdanDidukh2003) [inline app/src/main/java/com/bbdevs/app/MainActivity.kt:28]: two empty lines

**Review by nklymok — APPROVED** [review]

**Pull request merged**